src/components/details/TimestampWrapper.tsx

```diff
@@ -0,0 +1,82 @@
+import * as _ from 'lodash';
+import * as React from 'react';
+import { useTranslation } from 'react-i18next';
+
+import {
+  K8sModel,
+  K8sResourceCommon,
+  K8sResourceKindReference,
+  Timestamp,
+  useK8sModel,
+  useK8sWatchResources,
+} from '@openshift-console/dynamic-plugin-sdk';
+import { getReference } from '@openshift-console/dynamic-plugin-sdk/lib/utils/k8s/k8s-ref';
+
+import { GitOpsResource } from '../utils/gitops-types';
+
+interface TimestampWrapperProps {
+  resModels: GitOpsResource[];
+}
+
+export type K8sResourceKind = K8sResourceCommon & {
+  spec?: {
+    [key: string]: any;
+  };
+  status?: { [key: string]: any };
+  data?: { [key: string]: any };
+};
+
+export const getReferenceForModel = (model: K8sModel): K8sResourceKindReference =>
```

Comment:
Do we need to export this?

Reply:
Seems we can import it from SDK now, I will remove this export. Good catch 👍

Reply:
New Jira ticket created for tracking dynamic plugin SDK releases for replacing deprecated API: https://issues.redhat.com/browse/GITOPS-2316